src/sentry/conf/server.py

```diff
@@ -1472,3 +1477,12 @@ def get_sentry_sdk_config():
 SENTRY_USER_PERMISSIONS = (
     'broadcasts.admin',
 )
+
+KAFKA_PRODUCER_OPTIONS = {
+    'bootstrap.servers': 'localhost:9092',
+    'message.max.bytes': 50000000,  # 50MB, default is 1MB
+}
+
+KAFKA_PREPROCESS_TOPIC = 'preprocess-events'
+KAFKA_PROCESS_TOPIC = 'process-events'
+KAFKA_EVENT_SAVE_TOPIC = 'save-events'
```

Comment:
TODO: Need to be able to tie topics to their own clusters, so that in production we can produce to the correct one(s). Consumer side is already handed because we can provide whatever `--bootstrap-server` we want.